tensorflow/lite/python/lite.py

```diff
@@ -367,7 +368,7 @@ def from_keras_model_file(cls,
     """
     _keras.backend.clear_session()
     _keras.backend.set_learning_phase(False)
-    keras_model = _keras.models.load_model(model_file)
+    keras_model = _keras.models.load_model(model_file, custom_objects=custom_objects)
```

Comment:
And let this call be `_keras.models.load_model(model_file, **kwargs)`.
This is a way to make the API `from_keras_model_file()` somewhat future proof w.r.t underlying Keras `load_model()` API change?.